vMenuServer/vMenuServer.csproj

```diff
@@ -11,6 +11,14 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
     <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Please move this to the other config files further down in the file.
Also why is there 2 item groups, one that removes it, and another that includes it?